rust/src/dns/parser.rs

```diff
@@ -740,7 +868,7 @@ mod tests {
         ;; MSG SIZE  rcvd: 191   */
 
         let pkt: &[u8] = &[
-            0xeb, 0x56, 0x81, 0x80, 0x00, 0x01, 0x00, 0x02, 0x00, 0x00, 0x00, 0x01, 0x04, 0x5f,
+            0xeb, 0x56, 0x81, 0x80, 0x00, 0x01, 0x00, 0x02, 0x00, 0x00, 0x00, 0x00, 0x04, 0x5f,
```

Comment:
Hmm, curious how we ended putting a count of 1 here, when there is no data.

Reply:
yeah not sure on that, the dig output in the comment also shows `ADDITIONAL: 0`.